abbyy_to_epub3/create_epub.py

```diff
@@ -1123,12 +1135,22 @@ def craft_epub(self, epub_outfile="out.epub", tmpdir=None):
             epub_outfile = '%s.epub' % epub_outfile
         epub.write_epub(epub_outfile, self.book, {})
 
-        # run checks
-        verifier = EpubVerify(self.debug)
-        if self.args:
-            if self.args.epubcheck:
-                self.logger.info("Running EpubCheck on {}".format(epub_outfile))
-                verifier.run_epubcheck(epub_outfile)
-            if self.args.ace:
-                self.logger.info("Running DAISY Ace on {}".format(epub_outfile))
-                verifier.run_ace(epub_outfile)
+        # run validation on epub
+        if self.args and self.args.epubcheck:
+            self.validate_epub(epub_outfile)
+
+    def validate_epub(self, epub_file):
+        self.logger.debug("Running EpubCheck on {}".format(epub_file))
+        result = self.verifier.run_epubcheck(epub_file)
+        if not self.debug:
+            # we probably shouldn't hard code this list:
+            exceptions = ['Non-standard image resource of type image/x-ms-bmp found.']
+            errors = [err for err in result.messages if
+                      # filter out warnings (i.e. non errors) if not
+                      # in debug mode
+                      err.level.lower() == "error" and
+                      # filter our errors whose messages match known /
+                      # approved exceptions
+                      err.message not in exceptions]
```

Comment:
SUGGESTION: Maybe don't code so exactly to the entire line? e.g. just check for `image/x-ms-bmp found` present?